packaging/logstash/files/usr/lib/systemd/system/logstash.service.d/restart.conf

```diff
@@ -0,0 +1,3 @@
+[Service]
+Restart=always
```

Comment:
not adding restartsec or similar?

Reply:
ahh... this one slipped through the net. I'm going to now say we shouldn't do this change in the same PR as the RPM implementation, because that sounds like a good reason 👍